atlp-rwanda/winners-c8-bn-be
# #182210044 add Email verification feature

#### What does this PR do?
This PR adds a function (a helper) called `sendVerificationEmail()` to which can be passed an email address (string) as parameter, and send it an email message. This function will be used in sending verification email after a successful user registration.

#### Description of Task to be completed?
 Tasks to complete are:
 1. Create an email verification helper function `(in progress)`
 2. Integrate it into user registration route `(not yet done)`
#### How should this be manually tested?
Step 1. Clone this repository
Step 2. run `npm run install`
Step 3. configure the `.env` file in the root directory as the `.env.example` describes it.
Step 4. run `npm run dev`
Step 5. register through the route: `http://localhost:5000/api/auth/register/`
Step 6. check the inbox of the email you provided in `Step 5` and click on the sent link to verify your email

#### Any background context you want to provide?
N/A.
#### What are the relevant pivotal tracker stories?
`#182210044`
#### Screenshots (if appropriate)
The screenshot below shows how the email mesage will look like (HTML format)
![image](https://user-images.githubusercontent.com/83488648/172329051-6fc0d269-b9e9-44ad-9a8d-f73634ac0382.png)


#### Questions:
N/A
